docs/architecture_and_project_structure.md

```diff
@@ -1,7 +1,20 @@
 # Architecture
 
-> [!NOTE]
-> A description of the architectural decisions that were made on the project should be added here.
+![architecture scheme](/docs/images/architecture.png)
+
+### Presentation
+
+Logic in a presentation layer can be either business logic(Model) or UI logic(WidgetModel):
+* Business logic is the implementation of product requirements for application data. For example, bookmarking an article in a news reader app when a user clicks on a button.
+* UI logic is related to how to display UI state on the screen. For example, obtaining the right search bar hint when the user has selected a category, scrolling to a particular item in a list, or the navigation logic to a particular screen when the user clicks a button.
```

Comment:
I don't think we can take wording right from Android Developers website.
Change the examples used here, please.